files/docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Image for the web service (httpd), for celery worker see files/docker/Dockerfile.worker
 
-FROM docker.io/usercont/base
+FROM fedora:32
```

Comment:
you should update the [base image]( https://github.com/packit/deployment/blob/master/containers/Dockerfile.base)
it already contains ansible

Reply:
I know, this is the 'Temporary commit, to be removed' :-)

Reply:
ok sorry - got the point :)